### ISSUE TITLE
FR : Manage an existing elastic user, and periodically rotate it's password.

### ISSUE THREAD

**Issue by ninjadude333** [hashicorp/vault-plugin-database-elasticsearch#65]
Hi,
is it currently possible to add an existing elastic search user, and just configure periodic password rotation (same as the ability in LDAP secret engine static secret) ?

thanks,
David

**Comment by maxcoulombe** [comment]
Hi @ninjadude333 , yes the Elasticsearch Vault plugin supports rotating the password of existing users .

For database engines, rotating passwords for existing users is called `static roles`. Going through the doc, I noticed that while the [database capability table](https://developer.hashicorp.com/vault/docs/secrets/databases#database-capabilities) correctly reports this is available for the elasticsearch plugin there are no documented examples.

I opened a [PR to improve the doc](https://github.com/hashicorp/vault/pull/23077) so it is easier to discover.

Hope it answers your question! Cheers.
